etc/google-earth-pro.profile

```diff
@@ -0,0 +1,4 @@
+# Redirect
+include google-earth.profile
+
+private-bin google-earth-pro,sh,bash,grep,sed,ls,dirname
```

Comment:
you just need `private-bin google-earth-pro` since the rest are already included